InstallSpinnaker.sh

```diff
@@ -604,6 +638,17 @@ if [[ "${CLOUD_PROVIDER,,}" == "amazon" || "${CLOUD_PROVIDER,,}" == "google" ||
         write_default_value "AWS_VPC_ID" $AWS_VPC_ID
         write_default_value "AWS_SUBNET_ID" $AWS_SUBNET_ID
         ;;
+    all)
```

Comment:
I'm guessing at this point it would be simpler to have independent checks for each and have "both" set google and aws, "all" set each, and the explicit flags set its corresponding one. Then at the end iterate over each check and set as needed.